fastly/config_store.go

```diff
@@ -149,15 +149,28 @@ func (c *Client) GetConfigStoreMetadata(i *GetConfigStoreMetadataInput) (*Config
 	return csm, nil
 }
 
+// ListConfigStoreServicesInput is the input to ListConfigStoreServices.
+type ListConfigStoresInput struct {
+	// Name is the name of a config store (optional).
+	Name string
+}
+
 // ListConfigStores returns a list of config stores sorted by name.
-func (c *Client) ListConfigStores() ([]*ConfigStore, error) {
+func (c *Client) ListConfigStores(i *ListConfigStoresInput) ([]*ConfigStore, error) {
```

Comment:
This will result in a breaking interface change and a major release. Are we ok with doing that? If we're going to be pushing a breaking change we should check what else we can roll up into that release that is also breaking.